### PR TITLE
Bump Rust version to 1.69

### PR DESCRIPTION
**Pull request by danielsn**
# What does this PR do?

Uses the latest Rust version in Alpine.

# Motivation

Keeps up to date with the Rust version in Alpine

# Additional Notes

Anything else we should know when reviewing?

# How to test the change?

`cargo test` + CI

### CODE REVIEW

File: .gitlab-ci.yml
Comment (r1viollet) [inline .gitlab-ci.yml:5]: You can override these values when launching manual jobs within the gitlab pipeline (so you don't need to change this).

Comment (r1viollet) [reply .gitlab-ci.yml:5]: I can approve once you merge the downstream :+1: (as this will require a commit on your end to put main as default)

**Comment by danielsn** [comment]
Depends on https://github.com/DataDog/libddprof-build/pull/27

File: .gitlab-ci.yml
Comment (danielsn) [reply .gitlab-ci.yml]: @r1viollet downstream merged, and main set back as default

**Review by r1viollet — APPROVED** [review]
LGTM

**Pull request merged**
